kie-server-parent/kie-server-controller/kie-server-controller-rest/src/main/java/org/kie/server/controller/rest/RestRuntimeManagementServiceImpl.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.server.controller.rest;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.kie.server.controller.api.KieServerControllerException;
+import org.kie.server.controller.api.KieServerControllerIllegalArgumentException;
+import org.kie.server.controller.api.model.runtime.ContainerList;
+import org.kie.server.controller.api.model.runtime.ServerInstanceKeyList;
+import org.kie.server.controller.impl.service.RuntimeManagementServiceImpl;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.kie.server.controller.rest.ControllerUtils.*;
+
+@Path("/controller/runtime")
+public class RestRuntimeManagementServiceImpl extends RuntimeManagementServiceImpl {
+
+    private static final Logger logger = LoggerFactory.getLogger(RestRuntimeManagementServiceImpl.class);
+    private static final String REQUEST_FAILED_TOBE_PROCESSED = "Request failed to be processed due to: ";
+
+    @GET
+    @Path("servers/{id}")
```

Comment:
this endpoint should be servers/{id}/instances as this is what it actually returns - server instances for given template. At the moment it misleads that it will return server template identified by id

Reply:
Sure.